.gitmodules

```diff
@@ -0,0 +1,4 @@
+
+[submodule "kin-token"]
+	path = truffle/kin-token
+	url = https://github.com/kinfoundation/kin-token.git
```

Comment:
can we use subtree instead? subtree can be updated when needed in PR and effect all developer that just pull recent change - submodule require all developer to pull recent dev change plus remembering to update submodule

Reply:
@OrenZak I agree that it's best to change to subtree - @YohayBar this needs to be done for ios too ...

Reply:
@berryve @YohayBar  not sure on that, developers that would like only to run the sample app will also download kin-token submodule for nothing.